app/bundles/FormBundle/Security/Permissions/FormPermissions.php

```diff
@@ -7,13 +7,10 @@
 
 class FormPermissions extends AbstractPermissions
 {
-    /**
-     * @param array $params
-     */
-    public function __construct($params)
+    public function __construct(array $params)
     {
         parent::__construct($params);
-        $this->addCustomPermission('export', ['enable' => 1024]);
+        $this->addCustomPermission('export', ['enable' => 1024, 'notAnonymize' => 2]);
```

Comment:
Can we define constants for  these values: 1024, 2?

Reply:
I followed the convention that is already in effect, this is how all permissions are granted in Mautic